locustfile.py

```diff
@@ -13,7 +13,7 @@
 from lxml.html import submit_form
 from locust import HttpLocust, TaskSet, task
 
-MAX_UPLOAD_POLL_ATTEMPTS = 200
+MAX_UPLOAD_POLL_ATTEMPTS = 400
```

Comment:
hmm. I think 200 was ok before. Three minutes is way too long to wait for submission; I think we want to see these test failures so we can optimize.